tensorboard/components/tensor_widget/dtype-utils-test.ts

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+import {expect} from 'chai';
+
+import {isIntegerDType, isFloatDType} from './dtype-utils';
+
+describe('isIntegerDType', () => {
+  it('unsigned ints', () => {
```

Comment:
Please write a behavior here. We are using BDD style of testing but this seems more like a spec. 
In other words, it should read like English prose: "it returns whatever for foobar baz" instead of "it unsigned ints" doesn't make much sense.

Reply:
Done.